plugins/vpc-branch-eni/config/netconfig.go

```diff
@@ -50,25 +51,25 @@ type TAPConfig struct {
 // netConfigJSON defines the network configuration JSON file format for the vpc-branch-eni plugin.
 type netConfigJSON struct {
 	cniTypes.NetConf
-	TrunkName              string `json:"trunkName"`
-	TrunkMACAddress        string `json:"trunkMACAddress"`
-	BranchVlanID           string `json:"branchVlanID"`
-	BranchMACAddress       string `json:"branchMACAddress"`
-	BranchIPAddress        string `json:"branchIPAddress"`
-	BranchGatewayIPAddress string `json:"branchGatewayIPAddress"`
-	BlockIMDS              bool   `json:"blockInstanceMetadata"`
-	InterfaceType          string `json:"interfaceType"`
-	Uid                    string `json:"uid"`
-	Gid                    string `json:"gid"`
+	TrunkName          string   `json:"trunkName"`
+	TrunkMACAddress    string   `json:"trunkMACAddress"`
+	BranchVlanID       string   `json:"branchVlanID"`
+	BranchMACAddress   string   `json:"branchMACAddress"`
+	IPAddresses        []string `json:"ipAddresses"`
```

Comment:
the change of json tag will require a corresponding agent change due to the fact that the agent always build and use the plugin using latest commit of master in this repo

Reply:
Can you please confirm that the agent uses latest? I (maybe mistakenly) remember agent pulled a commit ID. If not, it should. Pulling "latest" in official builds is never a good idea.

Reply:
yes we are using the latest (maybe unintentionally since we specify commit in submodule). we will make a change to actually build with the commit of the submodule

Reply:
we've made the change to pin to specific commit